public/v4/apps/penpot.yml

```diff
@@ -66,15 +66,13 @@ services:
 
     # Frontend
     $$cap_appname:
+        image: penpotapp/frontend:$$cap_penpot_frontend_version
```

Comment:
Also, it's probably a good idea to use one version instead of 3:
- cap_penpot_frontend_version
- cap_penpot_backend_version
- cap_penpot_exporter_version

I assume all of them should always be the same?

Reply:
So I've bumped the default version to 1.18.4 as well as simplifying by using the same version throughout (and updated the label to reflect this). 

It could be that someone wants to run individual versions of each service but I think simple is better until that request is made.